docs/source/get_started/access_info/access_info_css.ipynb

```diff
@@ -0,0 +1,300 @@
+{
```

Comment:
<p>To match the qss version, I think there should be a markdown cell for: </p><p> </p><p>"If are interested in increasing your balance or have more information on your user role, please reach out to us at <a href="mailto:superstaq@infleqtion.com" target="_blank">superstaq@infleqtion.com</a> or join our <a href="https://join.slack.com/t/superstaq/shared_invite/zt-1wr6eok5j-fMwB7dPEWGG~5S474xGhxw" target="_blank">Slack workspace</a>."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SupertechLabs/client-superstaq/pull/543/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>